src/azure-cli/azure/cli/command_modules/monitor/operations/metric_alert.py

```diff
@@ -213,17 +218,41 @@ def _parse_action_removals(actions):
 
 
 def _parse_resource_type(scopes):
-    from msrestazure.tools import parse_resource_id
+    from azure.mgmt.core.tools import parse_resource_id
     from azure.cli.core import CLIError
```

Comment:
Should't `from knack.util import CLIError` ?

Reply:
It seems some codes use "from azure.cli.core.util import CLIError", so which one should we use?